README.md

```diff
@@ -205,67 +172,44 @@ Prerequisites:
   [Git for Windows][] includes Git Bash
   and tools which can be included in the global `PATH`.
 
-To build, launch a git shell (e.g. Cmd or PowerShell), run `vcbuild.bat`
-(to build with VS2017 you need to explicitly add a `vs2017` argument),
-which will checkout the GYP code into `build/gyp`, generate `uv.sln`
-as well as the necesery related project files, and start building.
-
-```console
-> vcbuild
-```
-
-Or:
-
-```console
-> vcbuild vs2017
-```
-
-To run the tests:
-
-```console
-> vcbuild test
-```
-
-To see all the options that could passed to `vcbuild`:
-
-```console
-> vcbuild help
-vcbuild.bat [debug/release] [test/bench] [clean] [noprojgen] [nobuild] [vs2017] [x86/x64] [static/shared]
-Examples:
-  vcbuild.bat              : builds debug build
-  vcbuild.bat test         : builds debug build and runs tests
-  vcbuild.bat release bench: builds release build and runs benchmarks
-```
-
+</details>
 
-### Unix
-
-For Debug builds (recommended) run:
+To build with autotools:
 
 ```bash
-$ ./gyp_uv.py -f make
-$ make -C out
+$ sh autogen.sh
+$ ./configure
+$ make
+$ make check
+$ make install
 ```
 
-For Release builds run:
+To build with [CMake][]:
 
 ```bash
-$ ./gyp_uv.py -f make
-$ BUILDTYPE=Release make -C out
-```
+$ mkdir -p build
 
-Run `./gyp_uv.py -f make -Dtarget_arch=x32` to build [x32][] binaries.
+$ (cd build && cmake .. -DBUILD_TESTING=ON) # generate project with tests
+$ cmake --build build -j 8                  # build (8 parallel jobs)
```

Comment:
Still having issues in my `Ubuntu-18.04` with `cmake version 3.10.2`. Apparently it does not recognize the `-j` option.
```sh
$ cmake --build build -j 8
Unknown argument -j
Unknown argument 8
```

The original instructions worked fine.

Reply:
`--parallel`/`-j` was introduced in cmake 3.12: https://cmake.org/cmake/help/v3.12/release/3.12.html#command-line
Our minimum cmake requirement is 3.4: https://github.com/libuv/libuv/blob/104255f5af57153a9c590a002f53cfb6c15d4dd0/CMakeLists.txt#L2